cplusplus.json

```diff
@@ -25,5 +25,119 @@
+  {"id": "cpp.libutp", "name": "libutp", "tech": ["internet"], "tags": ["BitTorrent"], "repo": "bittorrent/libutp",
+  "imports": ["utp.h"], "examples": ["https://github.com/bittorrent/libutp/blob/master/ucat.c"]},
+
+  {"id": "cpp.argh", "name": "Argh!", "tech": ["utils"], "tags": ["argument-parsing"], "repo": "adishavit/argh",
+  "imports": ["argh.h"], "examples": ["https://github.com/adishavit/argh/blob/master/example.cpp"]},
+  {"id": "cpp.args", "name": "args", "tech": ["utils"], "tags": ["argument-parsing"], "repo": "taywee/args",
+  "imports": ["args.hxx"], "examples": ["https://github.com/Taywee/args/tree/master/examples"]},
+  {"id": "cpp.boost-program_options", "name": "Boost.Program_options",  "tech": ["utils"], "tags": ["argument-parsing"], "repo": "boostorg/program_options",
+  "imports": ["boost/program_options.hpp"], "examples": ["https://github.com/boostorg/program_options/tree/develop/example"]},
+  {"id": "cpp.cli11", "name": "CLI11", "tech": ["utils"], "tags": ["argument-parsing"], "repo": "CLIUtils/CLI11",
+  "imports": ["CLI/CLI.hpp"], "examples": ["https://github.com/CLIUtils/CLI11/tree/master/examples"]},
+  {"id": "cpp.cxxopts", "name": "cxxopts", "tech": ["utils"], "tags": ["argument-parsing"], "repo": "jarro2783/cxxopts",
+  "imports": ["cxxopts.hpp"], "examples": ["https://github.com/jarro2783/cxxopts/tree/master/src"]},
+  {"id": "cpp.docopt.cpp", "name": "docopt.cpp",  "tech": ["utils"], "tags": ["argument-parsing"], "repo": "docopt/docopt.cpp",
+  "imports": ["docopt.h"], "examples": ["https://github.com/docopt/docopt.cpp/tree/master/examples"]},
+  {"id": "cpp.gflags", "name": "gflags", "tech": ["utils"], "tags": ["argument-parsing", "cflags"], "repo": "gflags/gflags",
+  "imports": ["gflags/gflags.h"], "examples": ["https://gflags.github.io/gflags/"]},
+  {"id": "cpp.ncurses", "name": "Ncurses", "tech": ["api"], "tags": ["gui", "framework"], "repo": "http://invisible-island.net/ncurses/",
+  "imports": ["ncurses_cfg.h", "curses.h"], "examples": ["http://invisible-island.net/ncurses/ncurses-examples.html"]},
+  {"id": "cpp.pdcurses", "name": "PDCurses", "tech": ["api"], "tags": ["gui", "framework"], "repo": "wmcbrine/PDCurses",
+  "imports": ["curses.h"], "examples": ["https://github.com/wmcbrine/PDCurses/tree/master/demos"]},
+  {"id": "cpp.replxx", "name": "replxx", "tech": ["cli"], "tags": ["readline", "interpreter"], "repo": "AmokHuginnsson/replxx",
+  "imports": ["replxx.hxx"], "examples": ["https://github.com/AmokHuginnsson/replxx/tree/master/examples"]},
+  {"id": "cpp.tclap", "name": "TCLAP", "tech": ["utils"], "tags": ["argument-parsing"], "repo": "http://tclap.sourceforge.net/",
+  "imports": ["tclap/CmdLine.h"], "examples": ["https://github.com/eile/tclap/tree/master/examples"]},
+  {"id": "cpp.termbox", "name": "Termbox", "tech": ["api"], "tags": ["gui", "framework"], "repo": "nsf/termbox",
```

Comment:
tech == gui, tags == framework

Please review all cases where you used tech == api, it should be relatively rare category.